lib/espresso-runner.js

```diff
@@ -66,29 +70,39 @@ class EspressoRunner {
   }
 
   async startSession (caps) {
-    let cmd = ['am', 'instrument', '-w', '-e', 'debug', process.env.ESPRESSO_JAVA_DEBUG === 'true' ? 'true' : 'false',
+    let cmd = ['shell', 'am', 'instrument', '-w', '-e', 'debug', process.env.ESPRESSO_JAVA_DEBUG === 'true' ? 'true' : 'false',
```

Comment:
It might be more readable to group some options, for example:
```
const cmd = [
  'shell'
  'am', 'instrument', 
  '-w',
   '-e', process.env.ESPRESSO_JAVA_DEBUG === 'true' ? 'true' : 'false'
...
]
```